pywinauto/controls/atspi_controls.py

```diff
@@ -278,3 +278,37 @@ def select(self, start=0, end=None):
 
         # return this control so that actions can be chained.
         return self
+
+
+class ImageWrapper(atspiwrapper.AtspiWrapper):
+
+    """Wrap image controls"""
+
+    _control_types = ['Image', 'Icon']
+
+    # -----------------------------------------------------------
+    def __init__(self, elem):
+        """Initialize the control"""
+        super(ImageWrapper, self).__init__(elem)
+
+    def get_description(self):
+        """Get image description"""
+        self.element_info.image_get_description()
+
+    def get_locale(self):
+        """Get image locale"""
+        self.element_info.image_get_locale()
+
+    def get_size(self):
+        """Get image size.
+        Return a tupe with width and height"""
```

Comment:
Typo: `type` -> `tuple`. Also empty line after sumary is necessary with wrapping closing quotes to next line.

Reply:
Resolved with e358ab2